aiida/orm/authinfos.py

```diff
@@ -130,21 +128,13 @@ def get_workdir(self):
         try:
             return self.get_metadata()[self.PROPERTY_WORKDIR]
         except KeyError:
-            return self.computer.get_workdir()
+            return self.computer.get_property('work_dir')
```

Comment:
The property seems to be called `workdir` - at least on profile (on your branch) I get:

```
In [1]: c=load_computer(1)

In [2]: c.get_metadata()
Out[2]:
{'shebang': '#!/bin/bash',
 'workdir': '/scratch/{username}/aiida_run/',
 'append_text': ' ',
 'prepend_text': ' ',
 'mpirun_command': ['srun', '-n', '{tot_num_mpiprocs}'],
 'default_mpiprocs_per_machine': 28}
```
would this be broken now?